### PR TITLE
Detect missing the --version flag in `dotnet tool install foo` invocations

### PR DESCRIPTION
**Pull request by realmarv**

### CODE REVIEW

**Comment by realmarv** [comment]
> Please rebase this PR pray Thanks

Done

**Comment by knocte** [comment]
@realmarv sorry I realised there were a couple of issues in this PR **after** I merged it, so please can you create a new PR that addresses them? I'm gonna point them out now.

File: src/FileConventions/Library.fs [hunk 1]
```diff
@@ -49,6 +49,24 @@ let DetectUnpinnedVersionsInGitHubCI(fileInfo: FileInfo) =
...
+    let fileLines = File.ReadLines fileInfo.FullName
+
+    let dotnetToolInstallRegex =
+        Regex("dotnet\\s+tool\\s+install\\s+", RegexOptions.Compiled)
+
+    let unpinnedDotnetToolInstallVersions =
+        fileLines
+        |> Seq.filter(fun line -> dotnetToolInstallRegex.IsMatch line)
+        |> Seq.filter(fun line ->
+            not(line.Contains("--version")) && not(line.Contains("-v"))
```
Comment (knocte) [inline src/FileConventions/Library.fs:64]: @realmarv line can have less parentheses (in particular, 4 chars less)

Comment (realmarv) [reply src/FileConventions/Library.fs:64]: https://github.com/nblockchain/conventions/pull/110

**Comment by realmarv** [comment]
> @realmarv sorry I realised there were a couple of issues in this PR **after** I merged it, so please can you create a new PR that addresses them? I'm gonna point them out now.

Here is the new PR: https://github.com/nblockchain/conventions/pull/110

**Pull request merged**